scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -280,7 +280,19 @@
         {{ wtf.form_field(form.chrom_pos) }}
       </div>
     </div>
+    <div class="col-2 offset-3">
+      <div id="hidecompounds" class="collapse">
+        {{ form.compound_rank_score.label(class="control-label", data_toggle="tooltip", data_placement="top", title="Shade compounds below this score as if dismissed") }}
+        {{ form.compound_rank_score(class="ml-1 form-control", onchange="this.form.submit()") }}
+      </div>
+    </div>
+    <div class="col-1">
+      <a class="btn btn-secondary btn-sm form-control" data-toggle="collapse" href="#hidecompounds" aria-expanded="false" aria-controls="hidecompounds">
+        Hide compounds
```

Comment:
Are the compounds hidden or shaded?

Reply:
😸 That depends! If the last compound on the variant compound badge list is shaded, the whole badge is hidden. The button could say either, the tooltip has both.